MyStack.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;  
+namespace Stack
+{
+    public class MyStack<T> : IMyStack<T>{
+        private List<T> stack;
+        public MyStack() 
+        {
+            this.stack = new List<T>();
+        }
+        
+        public void Push(T x) 
+        {
+            stack.Add(x);
+        }
+        
+        public T Pop() 
+        {
+            if(this.IsEmpty())
+            {
+                throw new InvalidOperationException("The stack is empty!");
+            }
+            T top = this.Peak();
+            stack.RemoveAt(stack.Count - 1);
+            return top;
+        }
+        
+        public T Peak() 
+        {
+            return stack[stack.Count - 1];
+        }
+        
+        public void Clear()
+        {
+            this.stack.Clear();//or pop till the stack become empty.
+        }
+        public bool IsEmpty()
+        {
+            return (stack.Count == 0);
+        }
+        public void Print()
+        {
+            if(this.IsEmpty())
+            { 
+                return;
+            }
+            T element = this.Peak();
+            Console.WriteLine(this.Pop());
+            this.Print();
+            this.Push(element);
+        }
+    }
+}
```

Comment:
new line

Reply:
usually you should have 1 class iin a file, 
but for methods 1 line between them is enough not sure If I got you right, 
and for this comment the new line is needed for the end of the file
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file